texar/modules/pretrained/bert_utils.py

```diff
@@ -169,35 +169,9 @@ def name_to_variable(model: nn.Module, name: str) -> nn.Module:
     return pointer
 
 
-def _default_download_dir() -> str:
-    r"""Return the directory to which packages will be downloaded by default.
-    """
-    package_dir = os.path.dirname(os.path.dirname(
-        os.path.dirname(os.path.dirname(__file__))))
-    if os.access(package_dir, os.W_OK):
-        texar_download_dir = os.path.join(package_dir, 'texar_download')
-    else:
-        # On Windows, use %APPDATA%
-        if sys.platform == 'win32' and 'APPDATA' in os.environ:
-            home_dir = os.environ['APPDATA']
-
-        # Otherwise, install in the user's home directory.
-        else:
-            home_dir = os.path.expanduser('~/')
-            if home_dir == '~/':
-                raise ValueError("Could not find a default download directory")
-
-        texar_download_dir = os.path.join(home_dir, 'texar_download')
-
-    if not os.path.exists(texar_download_dir):
-        os.mkdir(texar_download_dir)
-
-    return os.path.join(texar_download_dir, 'bert')
-
-
 def load_pretrained_bert(pretrained_model_name: str,
```

Comment:
Is it possible to make these utility functions staticmethods of respective pretrained classes? This way we can remove the `model_name` attribute and the if-checks.